olive/auto_optimizer/template_mapping.py

```diff
@@ -0,0 +1,36 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import logging
+from copy import deepcopy
+from pathlib import Path
+
+import yaml
+
+logger = logging.getLogger(__name__)
+
+TEMPLATE_CONFIG_PATH = Path(__file__).resolve().parent / "config_template"
+
+model_template_mapping = {
+    "bert": [["OnnxConversion", "OrtTransformersOptimization", "OnnxQuantization", "OrtPerfTuning"]],
+    "open_llama": [["OptimumConversion", "OrtTransformersOptimization", "OptimumMerging"]],
+}
```

Comment:
We should not need template mapping to determine if we need OnnxConversion or OptimumConversion.

Reply:
Same with <model>.yaml